source/push-to-deploy.md

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
Where is the information about GBs used in the build and how to change it?

Reply:
added